### DISCUSSION TITLE
How to specify Sample Symmetry when analyzing EBSD data using Mtex?

### DISCUSSION THREAD

**Discussion by Achintya-Patra** [mtex-toolbox/mtex#719] [Ask Anything] [answered]
I want to study the texture evolution of a rolled sample using EBSD data. When importing XRD data there is an option to include specimen symmetry but the same option doesn't appear when importing EBSD data. Thus, I want to know whether it takes Triclinic symmetry by default or somehow changes the symmetry to Orthorhombic.

**Comment by kilir** [comment, marked as answer]
Hi,
yes, EBSD data is inferred to have triclinic sample symmetry. May I ask why you would want to use a higher symmetry?
Cheers,
Rüdiger
